sycl/test-e2e/Matrix/joint_matrix_half_impl.hpp

```diff
@@ -24,7 +24,7 @@ void matrix_multiply(big_matrix<TResult, M, N> &C, big_matrix<T, M, K> &A,
      accessor accB{bufB, cgh};
      accessor accC{bufC, cgh};
 
-     cgh.parallel_for<mult<T, TM, TN, TK>>(
+     cgh.parallel_for<mult<TResult, TM, TN, TK>>(
```

Comment:
I guess, this line should also be updated accordingly: `size_t sg_size = get_sg_size<mult<T, TM, TN, TK>>(q);`